README.rst

```diff
@@ -124,7 +124,7 @@ filings, use the command:
 
 .. code-block:: console
 
-    $ xbrl_extract examples/ferc1-2021-sample.zip ./ferc1-2021-sample.sqlite \
+    $ xbrl_extract examples/ferc1-2021-sample.zip --db-path ./ferc1-2021-sample.sqlite \
```

Comment:
@zschira are you able to run this successfully? I get:

```
 File "./ferc-xbrl-extractor/src/ferc_xbrl_extractor/instance.py", line 425, in instances_from_zip
 filing["filename"]: datetime.datetime.fromisoformat(
    ~~~~~~^^^^^^^^^^^^
TypeError: string indices must be integers, not 'str'
```

Reply:
Did you do a `git pull`? I also had to update the example filings because they were the old format. When I run with the new filings everything works

Reply:
Oh silly, that did it. Sorry about that!